L/libjulia/common.jl

```diff
@@ -0,0 +1,162 @@
+function configure(version)
+    name = "libjulia"
+
+    checksums = Dict(
+        v"1.4.2" => "76a94e06e68fb99822e0876a37c2ed3873e9061e895ab826fd8c9fc7e2f52795",
+        v"1.5.1" => "1f138205772eb1e565f1d7ccd6f237be8a4d18713a3466e3b8d3a6aad6483fd9",
+    )
+    sources = [
+        ArchiveSource("https://github.com/JuliaLang/julia/releases/download/v$(version)/julia-$(version).tar.gz", checksums[version]),
+        DirectorySource("./bundled"),
+    ]
+
+    # Bash recipe for building across all platforms
+    script = raw"""
+    apk update
+    apk add coreutils libuv-dev utf8proc
+
+    cd $WORKSPACE/srcdir/julia*
+
+    # Apply patches
+    if [ -d $WORKSPACE/srcdir/patches ]; then
+    for f in $WORKSPACE/srcdir/patches/*.patch; do
+        echo "Applying path ${f}"
+        atomic_patch -p1 ${f}
+    done
+    fi
+
+    case ${target} in
+        *linux*)
+            OS=Linux
+        ;;
+        *mingw*)
+            OS=WINNT
+        ;;
+        *darwin*)
+            OS=Darwin
+        ;;
+        *freebsd*)
+            OS=FreeBSD
+        ;;
+    esac
+
+    cat << EOM >Make.host.user
+    override CC=/opt/x86_64-linux-musl/bin/x86_64-linux-musl-gcc
+    override CXX=/opt/x86_64-linux-musl/bin/x86_64-linux-musl-g++
+    override AR=/opt/x86_64-linux-musl/bin/x86_64-linux-musl-ar
+    USE_SYSTEM_LIBUV=1
+    USE_SYSTEM_UTF8PROC=1
+    # julia want's libuv.a
+    override LIBUV=/usr/lib/libuv.so
+    override LIBUTF8PROC=/usr/lib/libutf8proc.so.2
+    EOM
+
+    cat << EOM >Make.user
+    USE_SYSTEM_LLVM=1
+    USE_SYSTEM_LIBUNWIND=1
+    USE_SYSTEM_PCRE=1
+    USE_SYSTEM_OPENLIBM=1
+    USE_SYSTEM_DSFMT=1
+    USE_SYSTEM_BLAS=1
+    LIBBLASNAME=libopenblas
+    USE_SYSTEM_LAPACK=1
+    LIBLAPACKNAME=libopenblas
+    USE_SYSTEM_GMP=1
+    USE_SYSTEM_MPFR=1
+    USE_SYSTEM_SUITESPARSE=1
+    USE_SYSTEM_LIBUV=1
+    USE_SYSTEM_UTF8PROC=1
+    USE_SYSTEM_MBEDTLS=1
+    USE_SYSTEM_LIBSSH2=1
+    USE_SYSTEM_CURL=1
+    USE_SYSTEM_LIBGIT2=1
+    USE_SYSTEM_PATCHELF=1
+    USE_SYSTEM_ZLIB=1
+    USE_SYSTEM_P7ZIP=1
+
+    override XC_HOST=${target}
+    override OS=${OS}
+
+    #llvm-config-host is not available
+    override LLVMLINK=-L${prefix}/lib -lLLVM-9jl
+    override LLVM_CXXFLAGS=-I${prefix}/include # This is probably to simple
+    override LLVM_LDFLAGS=-L${prefix}/lib
+
+    # just nop this
+    override LLVM_CONFIG_HOST=
+
+    # julia expects libuv-julia.a
+    override LIBUV=${prefix}/lib/libuv.a
+
+    prefix=${prefix}
+    LOCALBASE=${prefix}
+    EOM
+
+    make BUILDING_HOST_TOOLS=1 NO_GIT=1 -j${nproc} -C src/flisp host/flisp
+    make clean -C src
+    make clean -C src/support
+    make clean -C src/flisp
+    make USE_CROSS_FLISP=1 NO_GIT=1 -j${nproc} julia-ui-release
+
+    # compile libjulia but don't try to build a sysimage
+    mkdir -p ${libdir}
+    mkdir -p ${includedir}/julia
+    cp usr/lib/libjulia* ${libdir}/
+    cp -R -L usr/include/julia/* ${includedir}/julia
+    install_license /usr/share/licenses/MIT
```

Comment:
Maybe use Julia's [license file](https://github.com/JuliaLang/julia/blob/f26a8c352fee56b845396f79152ccd01505d8973/LICENSE.md)?
```suggestion
    install_license LICENSE.md
```